falcon/typing.py

```diff
@@ -62,3 +65,30 @@
 Headers = Dict[str, str]
 HeaderList = Union[Headers, List[Tuple[str, str]]]
 ResponseStatus = Union[http.HTTPStatus, str, int]
+
+Resource = object
+
+
+class SyncResponderMethod(Protocol):
+    def __call__(
+        self,
+        resource: Resource,
+        req: wsgi.Request,
```

Comment:
Since `wsgi.Request` is only imported under `TYPE_CHECKING`, but this one is not, is this module even importable at all?

Reply:
If you have future-annotations all these are treated as strings, so it's not an issue